e2e-tests/cypress/support/check-header-service-name-href.js

```diff
@@ -0,0 +1,18 @@
+import { PRODUCT } from '../../content-strings';
+import header from '../e2e/partials/header';
+import { ROUTES } from '../../constants';
+
+const { ROOT: quoteStart } = ROUTES;
+const { START: insuranceStart } = ROUTES.INSURANCE;
+
+/**
+ * checks service name based on if isInsuranceRoute is true or false
+ * If true, then header should be linked to insuranceStart and have the application content string
+ */
+export default (isInsuranceRoute) => {
+  if (isInsuranceRoute) {
+    cy.checkLink(header.serviceName(), insuranceStart, PRODUCT.DESCRIPTION.APPLICATION);
+  } else {
+    cy.checkLink(header.serviceName(), quoteStart, PRODUCT.DESCRIPTION.QUOTE);
+  }
+};
```

Comment:
In any new commands can we define the params in an object and const declaration e.g

```js
const checkHeaderServiceNameHref = ({ isInsuranceRoute }) => {
  // ...
}

export default checkHeaderServiceNameHref;
```

The object params make it easier to update in the future as you can change a certain param without having to pass other params